### PR TITLE
Develop to next (Friday the 13th)

### PR DESCRIPTION
**Pull request by glennmatthews**
# Closes: #n/a
# What's Changed

Merge latest `develop` up to `next` and resolve the many, many merge conflicts.

**DO NOT SQUASH MERGE THIS**

### CODE REVIEW

File: nautobot/core/tests/test_utilities.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        # with self.subTest("Test skip non-UUID value display (legacy, ex: ModelMultipleChoiceFilter)"):
+        #     expected_output = {
+        #         "name": "manufacturer",
+        #         "display": "Manufacturer (slug)",
+        #         "values": [{"name": "fake_slug", "display": "fake_slug"}],
+        #     }
+
+        #     self.assertEqual(
+        #         check_filter_for_display(device_filter_set_filters, "manufacturer", ["fake_slug"]), expected_output
+        #     )
```
Comment (gsnider2195) [inline nautobot/core/tests/test_utilities.py:103]: Just want to verify that commenting these out was intended

File: nautobot/dcim/models/device_components.py [hunk 2]
```diff
@@ -980,6 +980,10 @@ def clean(self):
                     }
                 )
 
+    @property
+    def parent(self):
+        return self.device
+
```
Comment (gsnider2195) [inline nautobot/dcim/models/device_components.py:986]: I'm not seeing this in `DeviceBay` on `next` or `develop`

File: nautobot/core/tests/test_utilities.py (hunk 1 above)
Comment (glennmatthews) [reply nautobot/core/tests/test_utilities.py:103]: possibly circuit.termination_a/termination_z?

File: nautobot/dcim/models/device_components.py (hunk 2 above)
Comment (glennmatthews) [reply nautobot/dcim/models/device_components.py:986]: Was an incorrect removal in `next` - `develop` added a new test that exercises code that requires every device-component (as well as `VMInterface`) to have a `.parent` property. We didn't have this test in `next` before now so it was broken but we didn't know it.

**Review by gsnider2195 — APPROVED** [review]

**Pull request merged**